src/gui/syncrunfilelog.cpp

```diff
@@ -51,15 +51,19 @@ void SyncRunFileLog::start(const QString &folderPath)
         QFile::rename(filename, newFilename);
     }
     _file.reset(new QFile(filename));
-
     _file->open(QIODevice::WriteOnly | QIODevice::Append | QIODevice::Text);
-    _out.reset(new QDebug(_file.data()));
-    _out->noquote();
+
+
+    // we use a text stream to ensure the encoding is ok
+    // when outputiing info we use QDebug to ensure we can use the debug operatos
+    _out.reset(new QTextStream(_file.data()));
```

Comment:
By changing this from QDebug to QTextStream, don't you loose the formatting of standard types (QDateTime) that QDebug has? QDebug also takes an IODevice in the constructor which you could set an encoding before passing it on to QDebug.

For me this is a risky and not not really needed refactoring tbh. We really should not accidentially change any formatting of the file as people might parse it etc.

Reply:
hm, yes. Sorry.